### DISCUSSION TITLE
Can I let the song loop?

### DISCUSSION THREAD

**Discussion by SpaghettiLord1010** [cifkao/html-midi-player#8] [Q&A] [answered]
Sorry if this is a stupid question, but can I loop the song somehow?

I've tried to add the loop argument but it doesn't seem to work.
In any case: Thank you for making this!

**Comment by cifkao** [comment]
Currently there is no loop option. You can try listening to the [`stop` event](https://github.com/cifkao/html-midi-player/blob/master/doc/midi-player.md#events) in JavaScript and calling `start()` whenever it's triggered with `finished == true`. There may be some delay though.

**Reply by SpaghettiLord1010** [reply]
Eh, better than nothing, isn't it! Thank you for the quick answer, I'll try that! Have a great day!

**Comment by cifkao** [comment, marked as answer]
Looping is now implemented in v1.5.0! Just add the `loop` attribute to your player.
```html
<midi-player src="song.mid" loop></midi-player>
```

**Reply by SpaghettiLord1010** [reply]
Amazing! Thank you!
